src/components/LeafletMap.jsx

```diff
@@ -196,7 +194,7 @@ const LeafletMap = ({ showFilterPane, showMetricsPane, view, setView }) => {
       <MapAndTableControlsWrapper />
       <MarkerClusterGroup
         // TODO: Experiment with some of the "chunked" props to see if they improve performance: https://akursat.gitbook.io/marker-cluster/api
-        chunkedLoading
+        chunkedLoading={true}
```

Comment:
I think `chunkedLoading` was previously true. Just including the property makes it an implicit boolean which is interpreted as true. If you removed the prop it would be interpreted as false.

Have you looked at the other props in the docs? `chunkInterval` sounds promising.

Reply:
You're right. chunkedInterval seems to let the user user the filter while the markers are rendered. I didn't notice any difference with chunkedDelay though

Reply:
That sounds like a good performance improvement then 👍